src/stores/__tests__/index.cy.ts

```diff
@@ -13,4 +13,43 @@ describe("getPlayerTotalPoints()", () => {
     );
     expect(totalPoints).equals(0);
   });
+
+  it("IF PLAYER WITH THE MINIMUM POINTS SAID GABO", () => {
+    const map = new Map();
+    map.set("First player", { points: 3, saidGabo: true });
+    map.set("Second player", { points: 5, saidGabo: false });
+    const totalPoints = getPlayerTotalPoints(
+      { totalPoints: 3, name: "First player" },
+      map,
+      true,
+      () => {}
+    );
```

Comment:
do you think we can also test what the second player gained in the round? 

```suggestion
    const totalPointsFirstPlayer = getPlayerTotalPoints(
      { totalPoints: 3, name: "First player" },
      map,
      true,
      () => {}
    );
        const totalPointsSecondPlayer = getPlayerTotalPoints(
      { totalPoints: 3, name: "Second player" },
      map,
      true,
      () => {}
    );
```

And do the assertion (`expect`) of the second player.

Also, would be great to add one more player.

WHYT?